app/importers/applications_import.rb

```diff
@@ -0,0 +1,43 @@
+class ApplicationsImport
+  include ActiveModel::Model
+
+  attr_accessor :file
+
+  def initialize(attributes = {})
+    attributes.each { |name, value| send("#{name}=", value) } if !attributes.nil? && attributes.key?(:file)
+  end
+
+  def perform # rubocop:disable Metrics/MethodLength
+    if !file
+      errors.add :base, 'Field can\'t be empty!'
+      false
+    else
+      finding_errors
```

Comment:
again this check for errors should be redundant as it was for "Field can't be empty!"